components/TripleCounter/index.js

```diff
@@ -0,0 +1,134 @@
+import { useState, useEffect } from "react";
+import { useRouter } from "next/router";
+import { StyledButton } from "../../styles/StyledButton";
+import { StyledSpan } from "../../styles/StyledSpan";
+import { StyledInstruction2 } from "../../styles/StyledInstruction2";
+import { StyledRoundCounter } from "../../styles/StyledRoundCounter";
+
+export default function TripleCounter() {
+  const [tripleCounterOne, setTripleCounterOne] = useState(4);
+  const [tripleCounterTwo, setTripleCounterTwo] = useState(7);
+  const [tripleCounterThree, setTripleCounterThree] = useState(8);
+  const [isTripleCounterOneActive, setIsTripleCounterOneActive] =
+    useState(false);
+  const [isTripleCounterTwoActive, setIsTripleCounterTwoActive] =
+    useState(false);
+  const [isTripleCounterThreeActive, setIsTripleCounterThreeActive] =
+    useState(false);
+  const [tripleCounterOneIntervalId, setTripleCounterOneIntervalId] =
+    useState(null);
+  const [tripleCounterTwoIntervalId, setTripleCounterTwoIntervalId] =
+    useState(null);
+  const [tripleCounterThreeIntervalId, setTripleCounterThreeIntervalId] =
+    useState(null);
+  const [roundCounter, setRoundCounter] = useState(0);
+
+  const router = useRouter();
+
+  useEffect(() => {
```

Comment:
maybe you can find a way to make your code less repetitive here.

Reply:
Thanks for your advice. Actually that was exactly my plan for this weekend: Refactoring the code, cut components and make it easier to understand (as I already did with my styled components ;)).